### PR TITLE
Analytics 

### PR DESCRIPTION
**Pull request by singularityhacker**

### CODE REVIEW

**Comment by singularityhacker** [comment]
:wave: 0xJustice.eth here :white\_check\_mark:

File: pages/_app.tsx
Comment (geositta) [inline pages/_app.tsx:21]: please if you would confirm this is not sensitive data that should be placed in a .env file / enviroment config?

File: components/AnalyticsEventTracker.tsx
Comment (geositta) [inline components/AnalyticsEventTracker.tsx:20]: minor thing, spacing

File: pages/_app.tsx
Comment (zberwaldt) [inline pages/_app.tsx:28]: Do you want this console.log in?

File: components/MainArticle.tsx
Comment (geositta) [inline components/MainArticle.tsx:5]: future commits would be good to minimize unneeded whitespace.

File: pages/_app.tsx
Comment (zberwaldt) [reply pages/_app.tsx:28]: @singularityhacker

Comment (singularityhacker) [reply pages/_app.tsx:28]: It's fine. Not enough of a negative impact to deal with right now. We merely want an analysis on the site so we can compare it to the new site when we go live in two months.

File: components/AnalyticsEventTracker.tsx
Comment (singularityhacker) [reply components/AnalyticsEventTracker.tsx:20]: Yep. Will be mindful of that going forward.

File: components/MainArticle.tsx
Comment (singularityhacker) [reply components/MainArticle.tsx:5]: Yep. Will be mindful of that going forward.

File: components/AnalyticsEventTracker.tsx
Comment (roseja) [inline components/AnalyticsEventTracker.tsx:24]: nit: I'm not sure why this is commented out, but should we remove commented out code in the future and leave a `TODO:` if there is something we want to address later on?

File: components/Footer.tsx
Comment (roseja) [inline components/Footer.tsx:57]: This all looks a lot cleaner! +1

File: pages/_app.tsx
Comment (singularityhacker) [reply pages/_app.tsx:21]: @ken333135 Can you confirm we are good here? I'm sure it's not that big of a deal. It amounts to a google analytics tag.

File: components/Post.tsx
Comment (roseja) [inline components/Post.tsx:41]: nit: removing commented out code if we don't need it

File: pages/join.tsx
Comment (roseja) [inline pages/join.tsx:3]: Wanted to double check if this is being used anywhere else? The preview looked good to me on a quick smoke test.

File: pages/_app.tsx
Comment (roseja) [reply pages/_app.tsx:21]: Is there a difference between a sandbox and production account id? It looks like it's not super sensitive based on the code below, since it can be seen on the client side.

File: components/AnalyticsEventTracker.tsx
Comment (singularityhacker) [reply components/AnalyticsEventTracker.tsx:24]: Yep. Will be mindful of that going forward.

File: components/Footer.tsx
Comment (singularityhacker) [reply components/Footer.tsx:57]: :+1:

File: components/Post.tsx
Comment (singularityhacker) [reply components/Post.tsx:41]: :+1:

File: pages/join.tsx
Comment (singularityhacker) [reply pages/join.tsx:3]: I couldn't say. Just trying to get the analytics in.

File: pages/_app.tsx
Comment (roseja) [reply pages/_app.tsx:21]: I missed that the development ID is located below in a comment. I agree with @geositta that it might be helpful to have an environment config for this data in the future.

**Review by roseja — APPROVED** [review]
Approving after review of comments. However, I still think someone more familiar with this project should also give their sign off.

File: pages/_app.tsx
Comment (ken333135) [reply pages/_app.tsx:21]: yea the id is just like a google analytics tag. should not be an issue.
will take note to move this to a environment config file in the future!

**Comment by singularityhacker** [comment]
This analytics ticket is still not showing in prod. Can anyone advise?

**Comment by zberwaldt** [comment]
> This analytics ticket is still not showing in prod. Can anyone advise?

It looks like the deploy to vercel failed: https://github.com/BanklessDAO/bankless-dao-ghost-website/deployments/activity_log?environment=Production

**Comment by roseja** [comment]
There is a conversation in `#website-code` about the error. I don't have any kind of write access, and am guessing it's because I'm on a Guest Pass right now. The code wasn't deployed and the js error around dependencies needs to be resolved and merged in before the deploy can go out to production.

**Pull request merged**
